build/teamcity/cockroach/nightlies/pebble_nightly_build_test_binary.sh

```diff
@@ -45,7 +51,12 @@ bazel build --define gotags=bazel,invariants \
 cp $BAZEL_BIN/external/com_github_cockroachdb_pebble/internal/metamorphic/metamorphic_test_/metamorphic_test \
     "$DEST/$PEBBLE_SHA.test"
 chmod a+w "$DEST/$PEBBLE_SHA.test"
-echo "$PEBBLE_SHA"
 
-# Return DEPS.bzl and the patch to its previous contents.
-git checkout HEAD -- DEPS.bzl build/patches
+# Discard changes.
+git reset -q --hard HEAD
```

Comment:
Nit: I prefer `git checkout .` for this instead of `git reset -q --hard` (in the absence of some specific reason you must use the latter, which I don't think there is...)